### ISSUE TITLE
News

### ISSUE THREAD

**Issue by jbadeau** [sculptor/sculptor#218]
Hello,

I see that there is activity on this repo. Is there a plan to do a new release or any updates?

**Comment by tjuerge** [comment]
> Is there a plan to do a new release or any updates?

In the forum entry [New version of sculptor](https://groups.google.com/d/msg/sculptorgenerator/gnGAYGKqmZg/lmaGtOxEAQAJ) was asked who's interested in a new release. At this time the demand was sparse (1 response).

**Comment by tavoda** [comment]
Of course. I'm finishing some features but I expect in month we will make new 3.2.0 release. I'm investigating possibility to override DSLTransformation.
Some ideas for new feature?

**Comment by jbadeau** [comment]
I’m one of the original authors of the jhipster ide (xtext based). I was trying to figure out what it would take to add jhipster as a template.

**Comment by tjuerge** [comment]
> I was trying to figure out what it would take to add jhipster as a template.

[Sculptor's built-in templates](https://github.com/sculptor/sculptor/tree/develop/sculptor-generator/sculptor-generator-templates) can be overridden or extended by using [Sculptor's extension mechanism](http://sculptorgenerator.org/documentation/advanced-tutorial#extension). Those extensions are distributed as [Sculptor cartridges](http://sculptorgenerator.org/documentation/advanced-tutorial#cartridges). Examples can be found in [Sculptor's cartridges aggregator project](https://github.com/sculptor/sculptor/tree/develop/sculptor-cartridges).
